runtime/src/lib.rs

```diff
@@ -364,12 +383,15 @@ cfg_if::cfg_if! {
                     | Call::Utility(_)
                     | Call::Vesting(_) => true,
 
+                    #[cfg(not(feature = "without-sudo"))]
+                    Call::Sudo(_) => true,
```

Comment:
Maybe depend on the compiler flag check feature [here](https://github.com/zeitgeistpm/zeitgeist/blob/f3db56ff8945ffef19e696bc8d6c026b753c2a39/runtime/src/lib.rs#L419) too.

Reply:
We don't need it in the last case, because when sudo is included, it falls into the `_ => ` match arm and calls won't be filtered. When it's not included it is not regarded at all.

Reply:
I understand, you mean because of not including the whole `Sudo` pallet in the `construct_runtime`. That makes sense.